keyboards/ramlord/witf/keymaps/default/keymap.c

```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
```

Comment:
Needs compliant licence header

```suggestion
// Copyright 2022 Sleepdealer (@Sleepdealr)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```